src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000029/TA-000029.json

```diff
@@ -0,0 +1,160 @@
+[
+  {
+    "Template": "APIMService.json",
+    "ReportedFailures": [
+      /*
+      //This is the real list of ReportedFailures. Will update file to this comment once repeated ReportedFailure bug is resolved
+
+      {
+        "LineNumber": 130,
+        "Description": "protocols contains http"
+      },
+      {
+        "LineNumber": 149,
+        "Description": "protocols contains ws"
+      },
+      {
+        "LineNumber": 168,
+        "Description": "protocols contains http and ws"
+      },
+      {
+        "LineNumber": 188,
+        "Description": "protocols contains http"
```

Comment:
Yes, but if we don't mention something like this then it looks like there are multiple resources that test the exact same test case, the description should be unique for each line number (except for the repeated ones because of what we already discussed)